ssh/tailssh/tailssh_test.go

```diff
@@ -245,8 +243,8 @@ func TestSSH(t *testing.T) {
 	}
 
 	t.Run("env", func(t *testing.T) {
-		cmd := execSSH("env")
-		cmd.Env = append(os.Environ(), "LANG=foo")
+		cmd := execSSH("LANG=foo env")
```

Comment:
how's this work?

Reply:
i don't understand the question. We should not be passing in any local env variables over SSH, however we also want to allow users to be able to specify env variables before command they run.